src/legacy/core_plugins/kibana/public/management/index.js

```diff
@@ -46,6 +54,78 @@ require('ui/index_patterns/route_setup/load_default')({
   whenMissingRedirectTo: '/management/kibana/index'
 });
 
+export function updateLandingPage(version) {
+  const node = document.getElementById(LANDING_ID);
+  if (!node) {
+    return;
+  }
+
+  render(
+    <EuiPageContent verticalPosition="center"  horizontalPosition="center">
+      <I18nProvider>
+        <div>
+          <div className="eui-textCenter">
+            <EuiIcon type="managementApp" size="xxl" />
+            <EuiSpacer />
+            <EuiTitle>
+              <h1>
+                <FormattedMessage
+                  id="kbn.management.landing.header"
+                  defaultMessage="Kibana {version} management"
+                  values={{ version }}
+                />
+              </h1>
+            </EuiTitle>
+            <EuiText>
+              <FormattedMessage
+                id="kbn.management.landing.subhead"
+                defaultMessage="Manage your data, permissions, and content from an administrator level"
```

Comment:
@lukeelmers Working off the description above  (and the fact that a lot more has been added to the Management app since 6.3), I'm thinking of something like this:  Manage your indices, index patterns, saved objects, Kibana settings, and more.